docs/resources/elb_active_standby_pool.md

```diff
@@ -0,0 +1,279 @@
+  + **slave**: standby backend server.
+
+* `name` - (Optional, String, ForceNew) Specifies the backend server name.
+
+* `subnet_cidr_id` - (Optional, String, ForceNew) Specifies the ID of the IPv4 or IPv6 subnet where the backend server
+  resides.
+
+  -> **NOTE:** 1. The IPv4 or IPv6 subnet must be in the same VPC as the subnet of the load balancer. 2.If this parameter
+  is not passed, IP as a Backend has been enabled for the load balancer. In this case, IP as backend servers must use
+  private IPv4 addresses, and the protocol of the backend server group must be TCP, HTTP, or HTTPS.
+
+<a name="healthmonitor"></a>
+The `healthmonitor` block supports:
+
+* `delay` - (Required, Int, ForceNew) Specifies the interval between health checks, in seconds.
+
+* `max_retries` - (Required, Int, ForceNew) Specifies the number of consecutive health checks when the health check
+  result of a backend server changes from OFFLINE to ONLINE.
+
+* `timeout` - (Required, Int, ForceNew) Specifies the maximum time required for waiting for a response from the health
+  check, in seconds. It is recommended that you set the value less than that of parameter delay.
+
+* `type` - (Required, String, ForceNew) Specifies the health check protocol. Value options: **TCP**, **UDP_CONNECT**,
+  **HTTP**, and **HTTPS**.
+
+* `domain_name` - (Optional, String, ForceNew) Specifies the domain name that HTTP requests are sent to during the health
+  check.
+
+* `expected_codes` - (Optional, String, ForceNew)  Specifies the expected HTTP status code. This parameter will take
+  effect only when type is set to HTTP or HTTPS.
+
+* `max_retries_down` - (Optional, Int, ForceNew) Specifies the number of consecutive health checks when the health check
+  result of a backend server changes from ONLINE to OFFLINE.
+
+* `monitor_port` - (Optional, Int, ForceNew) Specifies the port used for the health check. If this parameter is left
+  blank, a port of the backend server will be used by default.
+
+* `name` - (Optional, String, ForceNew) Specifies the health check name.
+
+* `url_path` - (Optional, String, ForceNew) Specifies the HTTP request path for the health check. The value must start
+  with a slash (/), and the default value is /. The value can contain letters, digits, hyphens (-), slashes (/),
+  periods (.), percentage signs (%), question marks (?), pound signs (#), ampersand signs (&), and the extended character
+  set _;~!()*[]@$^:',+.
+
+  -> **NOTE:** This parameter is available only when type is set to HTTP or HTTPS.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The unique ID for the pool.
+
+* `loadbalancers` - The IDs of the load balancers with which the backend server group is associated.
+  The [object](#loadbalancers) structure is documented below.
+
+* `listeners` - The IDs of listeners with which the backend server group is associated.
+  The [object](#listeners) structure is documented below.
```

Comment:
it is useless, fix them like the resource of huaweicloud_elb_member

Reply:
done